src/utils/helpers/size-to-string.ts

```diff
@@ -0,0 +1,4 @@
+export const sizeToString = (size: number | string | undefined): string | undefined => {
```

Comment:
you can set the argument as `size?: number | string` instead of `number | string | undefined` it the same but shorter 😉